src/FlightMap/MapItems/CameraTriggerIndicator.qml

```diff
@@ -22,7 +22,7 @@ MapQuickItem {
 
     sourceItem: Rectangle {
         width:      _radius * 2
-        height:     _radius * 2
+        height:     _radius * 2 * 1.5  // Increased height by 50%
```

Comment:
I don't think we ever show the camera trigger, do we need this code? Maybe your settings are non-standard for development, should use the same as ops

Reply:
Yeah this should not be there, removed it